app/models/wiki.rb

```diff
@@ -110,7 +115,9 @@ def self.titleize(title)
   end
 
   def create_menu_item_for_start_page
-    wiki_menu_item = wiki_menu_items.find_or_initialize_by_title start_page, name: 'Wiki'
+    wiki_menu_item = wiki_menu_items.find_or_initialize_by(title: start_page) { |item|
+      item.name = 'Wiki'
```

Comment:
I am aware that this is not introduced in this PR but why don't we use I18n here?